### ISSUE TITLE
Add openal verb to support recent built-in removal of OpenAL from WINE

### ISSUE THREAD

**Issue by jm2** [Winetricks/winetricks#2011]
As of WINE 7.21, OpenAL is no longer included as a built-in library[1]. As a result, it'd be very helpful if winetricks supported a standalone OpenAL library installation from openal.org as a verb. I'm hoping to have a pull request ready shortly to support this change.

[1] https://gitlab.winehq.org/wine/wine/-/commit/45eca854c4fab7e864deebf9a51c74147884ebc8

**Comment by Gcenx** [comment]
The entire reason for removing openal from wine is it’s not part of windows and anything that requires openal would already be required to ship it.

Do you happen to have examples of applications/games that require native openal but don’t ship with the dlls?

**Comment by jm2** [comment]
Yeah I understand the reason; unfortunately I was unknowingly depending on built-in OpenAL for some of my older games. The games it was impacting for me are GRID (2007), DiRT 2, DiRT 3, and DiRT Showdown. I imagine the Steam first-launch setup may install OpenAL silently (or maybe an alternative like Rapture3D) for these games, but I was downloading them via steamcmd without using the redistributables. In the case of GRID it came from GOG, but it's probably also the case where OpenAL was included in a redistributable folder elsewhere and run by the official installer.

Long story short: I wanted to avoid installing bundled game redistributables where possible, as historically they tend to be unnecessary and unpredictable with WINE except in certain specific circumstances.

**Comment by austin987** [comment]
Fixed by 61f4de17e3848ffe29cd0936f0e546a125c5b8d5.
